### PR TITLE
Lab 3

### PR DESCRIPTION
**Pull request by polinasevko**

### CODE REVIEW

**Comment by nastik1811** [comment]
And where is the link to deployed app?..

**Comment by polinasevko** [comment]
Screenshot (60)
Here

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      #message h1 { font-size: 22px; font-weight: 300; color: rgba(0,0,0,0.6); margin: 0 0 16px;}
+      #message p { line-height: 140%; margin: 16px 0 24px; font-size: 14px; }
+      #message a { display: block; text-align: center; background: #039be5; text-transform: uppercase; text-decoration: none; color: white; padding: 16px; border-radius: 4px; }
+      #message, #message a { box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24); }
+      #load { color: rgba(0,0,0,0.4); text-align: center; font-size: 13px; }
+      @media (max-width: 600px) {
+        body, #message { margin-top: 0; background: white; box-shadow: none; }
+        body { border-top: 16px solid #ffa100; }
+      }
+    </style>
```
Comment (nastik1811) [inline 404.html]: all styles should be in css files

**Review by nastik1811 — APPROVED** [review]
ok, it's `8`

**Pull request merged**